core/server/adapters/scheduling/post-scheduling/index.js

```diff
@@ -41,6 +41,7 @@ _private.loadClient = function loadClient() {
  * @return {Promise}
  */
 _private.loadScheduledPosts = function () {
+    // TODO: make this version aware?
```

Comment:
Probably a good idea to use v2 as default here so there are no surprises when its automatically switched when `v0.1` is gone? 🤔

Reply:
@rishabhgrg this bit will have to come into play as a follow-up change/issue (possibly in the next cycle). The main reason being there's a lot more to it then just adding `v2` here :) We'll need to migrate auth for default scheduler, make sure it handles 2 different endpoints for fetching entries  (posts/pages) and probably some more that I'm overlooking atm.

Reply:
Ah yeah, good point. Def didn't intend it as part of this PR though, more general of keeping in mind of everywhere API defaults to `0.1` right now and remember to handle it before we switch next cycle 😄